drivers/volume/portworx/portworx.go

```diff
@@ -854,6 +857,18 @@ func (p *portworx) FindSnapshot(tags *map[string]string) (*crdv1.VolumeSnapshotD
 	return nil, nil, &errors.ErrNotImplemented{}
 }
 
+func (p *portworx) GetSnapshotType(snap *crdv1.VolumeSnapshot) (string, error) {
+	// TODO: Check if is a portworx snapshot
+	snapType, err := getSnapshotType(snap)
+	if err != nil {
+		return "", err
+	}
+	if isGroupSnap(snap) {
+		return "Group " + string(snapType), nil
```

Comment:
nit: Use lower case group as for other snapTypes, it's "local" and "cloud". So the display will look odd if some are upper case and some lower.

Reply:
Updated